parm/config/config.fcst

```diff
@@ -133,13 +172,45 @@ elif [ $imp_physics -eq 11 ]; then # GFDL
     export hord_xx_nh_nonmono=5
     export vtdm4_nh_nonmono=0.02
     export nord=2
-    export dddmp=0.1
     export d4_bg=0.12
+    export dddmp=0.1
 
 else
     echo "Unknown microphysics option, ABORT!"
 
 fi
+
+#coupling settings
+export FRAC_GRID=".true."
+export cplmode="nems_frac"
+export psm_bc="1"
+
+export min_lakeice="0.15"
+export min_seaice="1e-11"
+
+export FSICL="99999"
+
+# Stochastic physics parameters
+# nam_stochy section
+export SEEDLET="10"
+export DO_SKEB="YES"
```

Comment:
Stochastic physics should be off until we start dealing with ensembles.

Reply:
Updated to set them to NO. Thanks!